lte/gateway/c/oai/tasks/nas/emm/Attach.c

```diff
@@ -2473,17 +2473,15 @@ void proc_new_attach_req(struct ue_mm_context_s* ue_context_p) {
       LOG_NAS_EMM,
       "Process new Attach Request for ue_id " MME_UE_S1AP_ID_FMT "\n",
       ue_context_p->mme_ue_s1ap_id);
-  new_attach_info_t* attach_info = ue_context_p->emm_context.new_attach_info;
-  if (!attach_info) {
-    OAILOG_ERROR_UE(
-        LOG_NAS_EMM, ue_context_p->emm_context._imsi64,
-        "New attach request within EMM context is null \n");
-    OAILOG_FUNC_OUT(LOG_NAS_EMM);
-  }
+  new_attach_info_t attach_info = {0};
+  memcpy(
+      &attach_info, ue_context_p->emm_context.new_attach_info,
+      sizeof(new_attach_info_t));
+  free_wrapper((void**) &ue_context_p->emm_context.new_attach_info);
```

Comment:
this looks problematic to me as there is no check whether new_attach_info is null or not.

Reply:
looks like we have the same changes on master...

Reply:
@pruthvihebbani can you double check to see if new_attach_info gets freed up when ue context is removed?

Reply:
I see on the master that the caller actually makes the check for `new_attach_info`. Maybe then we should add a comment above the function that is the case.

Reply:
> @pruthvihebbani can you double check to see if new_attach_info gets freed up when ue context is removed?

Currently new_attach_info is not freed up when UE context is removed. So we need to add null check and then free new_attach_info